src/seedu/addressbook/commands/Hello.java

```diff
@@ -0,0 +1,16 @@
+package seedu.addressbook.commands;
+
+
+public class Hello extends Command {
```

Comment:
Could name the class "HelloCommand" instead, following the format of the classes for the other commands!